packages/service-provider-server/src/cli-service-provider.ts

```diff
@@ -1210,25 +1210,30 @@ class CliServiceProvider extends ServiceProviderCore implements ServiceProvider
   getSearchIndexes(
     database: string,
     collection: string,
+    indexName?: string,
     // TODO(MONGOSH-1471): use ListSearchIndexesOptions once available
-    options: any,
+    options?: Document,
     dbOptions?: DbOptions): Promise<Document[]> {
     return this.db(database, dbOptions)
       .collection(collection)
       // @ts-expect-error still @internal
-      .listSearchIndexes(options).toArray();
+      .listSearchIndexes(indexName, options).toArray();
   }
 
   createSearchIndexes(
     database: string,
     collection: string,
     // TODO(MONGOSH-1471): use SearchIndexDescription[] once available
-    descriptions: any[],
+    specs: {name: string, definition: Document}[],
     dbOptions?: DbOptions): Promise<string[]> {
+    const driverSpecs = specs.map(({ name, definition }) => {
+      // NOTE: Unlike in our docs or in the command, the driver calls it description, not definition
```

Comment:
https://github.com/mongodb/node-mongodb-native/pull/3672/files#diff-7ce3e491e925915c35caff51955ce54f08a866b6a7c51e7c3daa2c0c6b0a848dR15

Reply:
Actually I'm fairly sure that's a bug on the driver. Will ping them.

Reply:
Yeah, it’s `definition`: https://github.com/10gen/mongo-enterprise-modules/blob/adaa6e278c12045b70c70d53a3b6bb957472f2ab/src/search/search_index_commands.idl#L57

I think in the driver that doesn’t really affect anything, though, it’s just variable naming